content/en/logs/explorer/calculate_queries.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Calculate Queries
+kind: documentation
+disable_toc: false
+private: true
+---
+
+## Overview
+
+Use calculated fields when you want to apply boolean logic, perform arithmetic operations, and/or manipulate strings while querying your logs.
```

Comment:
```suggestion
Use calculated fields when you want to apply Boolean logic, perform arithmetic operations, and/or manipulate strings while querying your logs.
```

Reply:
I'm actually seeing it lowercased in our docs.